File: redsheet-portal/src/app/database-models/v-lite-our-team-member.ts
import { required, maxLength, range, nested } from '@rx/annotations';

export class vLiteOurTeamMember {
    constructor(vLiteOurTeamMember?: vLiteOurTeamMember )  {
        let properties = [ "liteOurTeamMemberId", "liteOurTeamMemberName", "liteProjectBackgroundId", "personalityId", "personalityKeyText", "position",];
        for (let property of properties)
            if (vLiteOurTeamMember && vLiteOurTeamMember[property])
                this[property] = vLiteOurTeamMember[property];
    }
 
	liteOurTeamMemberId : number =   0 ;
 
    @required()
    @maxLength(50)
	liteOurTeamMemberName : string =   undefined;
 
    @range(1,2147483647)
	liteProjectBackgroundId : number =   undefined;
 
    @range(1,2147483647)
	personalityId : number =   undefined;
 
    @maxLength(10)
	personalityKeyText : string =   undefined;
 
    @required()
    @maxLength(50)
	position : string =   undefined;


}

File: RedSheet.DbEntities/Models/vLiteOurTeamMember.cs

using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations.Schema;
using System.ComponentModel.DataAnnotations;
using Microsoft.EntityFrameworkCore;
using Rx.Core.Data.Attributes;
using RedSheet.DbEntities.Enums;

namespace RedSheet.DbEntities.Models
{
	[Table("vLiteOurTeamMembers", Schema = "dbo")]
	[Serializable]
	public partial class vLiteOurTeamMember  
	{
 
	[DatabaseGenerated(DatabaseGeneratedOption.None)] 
	[Key]
	public int LiteOurTeamMemberId { get; set; } 
 
	[Required] 
	[MaxLength(50)]
	public string LiteOurTeamMemberName { get; set; } 
 
	[Range(1, int.MaxValue)]
	public int LiteProjectBackgroundId { get; set; } 
 
	[Range(1, int.MaxValue)]
	public int PersonalityId { get; set; } 
 
	[MaxLength(10)]
	public string PersonalityKeyText { get; set; } 
 
	[Required] 
	[MaxLength(50)]
	public string Position { get; set; } 

	public vLiteOurTeamMember()
	{
    }

    }
}

